### PR TITLE
deprecation(engine): event_api method remove title parameter, use summary instead

### LINKED ISSUE #26: deprecation(engine): event_api method remove `title` parameter, use `summary` instead
Starting hopeit.engine 0.2.0 version `title` parameter will be remove from event_api method, use summary instead.

- [x] make `event_api` named parameter method and remove `title` parameter.
- [ ] bump version.
- [ ] add removal note to release notes.

Related #25

### PR DESCRIPTION
**Pull request by pcanto-hopeit**
SPEC:
Closes #26

RELEASE NOTES
Removed `title` parameter in `event_api` method. Use `summary` instead.

### CODE REVIEW

**Review by leosmerling-hopeit — APPROVED** [review]

**Review by leosmerling-hopeit — CHANGES_REQUESTED** [review]
Please increase version to 0.2.0 for both engine, plugins and apps.

**Review by pcanto-hopeit — COMMENTED** [review]
@leosmerling-hopeit In the engine I found that the file where the engine version is defined globally was decoupled from the package version, so I made the setup.py get the package version from the hopeit.server.version.
I also updated the simple-example and basic-auth versions and their respective tests.

File: plugins/auth/basic-auth/setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setuptools.setup(
     name="hopeit.plugins.basic_auth",
-    version="0.1.0",
+    version="0.make qa.0",
```
Comment (leosmerling-hopeit) [inline plugins/auth/basic-auth/setup.py:6]: 0.2.0

File: engine/setup.py [hunk 2]
```diff
@@ -11,14 +15,13 @@ def read_requirements_lock():
...
 
-
 def libversion(lib):
     return versions[lib.split('[')[0]]
 
 
 setuptools.setup(
     name="hopeit.engine",
-    version="0.1.5",
+    version=version['ENGINE_VERSION'],
```
Comment (leosmerling-hopeit) [inline engine/setup.py:24]: :+1:

**Review by leosmerling-hopeit — CHANGES_REQUESTED** [review]

File: plugins/auth/basic-auth/setup.py (hunk 1 above)
Comment (pcanto-hopeit) [reply plugins/auth/basic-auth/setup.py]: fix on 7cd0cff27ef6aef597a194b6a7106fd2977bd077

**Review by leosmerling-hopeit — APPROVED** [review]

**Pull request merged**